server/index.js

```diff
@@ -4,14 +4,15 @@ import bodyParser from 'body-parser';
 import cors from 'cors';
 import passport from 'passport';
 import cookieSession from 'cookie-session';
-import passportSetup from './configs/passport-setup';
+import './configs/passport-setup';
```

Comment:
only after including this the passport object can associate with google authentication, could have included the code directly in index.js file but it would have cluttered that file.